lib/hexpm/accounts/session.ex

```diff
@@ -0,0 +1,21 @@
+defmodule Hexpm.Accounts.Session do
```

Comment:
Yeah, introducing Sessions context sounds like the way to go; or we could finally try writing Accounts one. We can do it after this lands, especially if we'll go with the latter